src/sidebar/helpers/groups.js

```diff
@@ -84,6 +84,10 @@ function isScopedToUri(group, uri) {
   return true;
 }
 
+/**
+ * @param {string} uri
+ * @param {string[]} scopes
```

Comment:
At some point to alleviate developer confusion it might be nice to type `scopes` in terms of what they actually _are_ and _mean_, but this is fine for here!